lib/base/error-handler.js

```diff
@@ -84,12 +84,13 @@ const log = require('npmlog'),
             "INVALID_CSV_FORMAT" : "Please change the file extension to .csv",
             "INVALID_COMMAND" : "This command is invalid. Please check the supported commands using ares -l",
             "INVALID_COMBINATION" : "This options cannot be used with display option",
-            
+
             // set value & type error
             "EMPTY_PROFILE" : "Profile is empty",
             "EMPTY_VALUE" : "Please specify a value",
             "EXISTING_VALUE" : "The specified value already exists",
             "EXISTING_FILETYPE_PATH" : "A file with the same name at the specified location already exists",
+            "SUPPORT_ONLY_IPK" : "Support only .ipk file. Please check the file extension",
```

Comment:
Why don't you change this error message? 
Only .ipk file is supported.

Please refer to : 
"NOT_SUPPORT_XZ" : "xz file is not supported. Only .ipk file packaged with the CLI is supported",